.env.example

```diff
@@ -31,6 +31,9 @@ DATABASE_URL="postgresql://postgres:@localhost:5450/calendso"
 # Use the same one as DATABASE_URL if you're not using a connection pooler
 DATABASE_DIRECT_URL="postgresql://postgres:@localhost:5450/calendso"
 INSIGHTS_DATABASE_URL=
+# Optionnal. In some cases it might make sense (e.g. when creating and dropping databases is not allowed on cloud-hosted databases) to manually define the connection string and name 
```

Comment:
```suggestion
# Optional. In some cases it might make sense (e.g. when creating and dropping databases is not allowed on cloud-hosted databases) to manually define the connection string and name 
```